driver/src/main/java/org/neo4j/driver/Config.java

```diff
@@ -345,7 +345,7 @@ public static final class ConfigBuilder {
         private long idleTimeBeforeConnectionTest = PoolSettings.DEFAULT_IDLE_TIME_BEFORE_CONNECTION_TEST;
         private long maxConnectionLifetimeMillis = PoolSettings.DEFAULT_MAX_CONNECTION_LIFETIME;
         private long connectionAcquisitionTimeoutMillis = PoolSettings.DEFAULT_CONNECTION_ACQUISITION_TIMEOUT;
-        private String userAgent = format("neo4j-java/%s", driverVersion());
+        private String userAgent = driverVersion();
```

Comment:
Shouldn't it keep as `format("neo4j-java/%s", driverVersion())`? afaik, the userAgent should not changed by the `boltAgent` introduction since the user can have integrations depending on this log.

Reply:
Oh, I was not actually going to change that. It is a side-effect of reworking this PR. Thanks for noticing this. Fixed.